### PR TITLE
Add gemmi recipe

### PR DESCRIPTION
**Pull request by mcs07**

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
I think that should be in ``bioconda's`` channel
What do you think @bgruening

**Comment by mcs07** [comment]
I added `{{ compiler('c') }}` after seeing the Windows build fail on Azure, but on second thought that is probably not needed. Probably best to wait and see if AppVeyor succeeds.

File: recipes/gemmi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - cmake
+    - ninja  # [win]
+  host:
+    - python
+    - pip
+    - pybind11
+    - zlib
+  run: 
+    - python
+    - zlib
```
Comment (bgruening) [inline recipes/gemmi/meta.yaml]: zlib can be removed here

File: recipes/gemmi/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cd build
+
+cmake .. \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release
+
+make -j$CPU_COOUNT
+
+cp gemmi "$PREFIX/bin/"
```
Comment (bgruening) [inline recipes/gemmi/build.sh]: Why is this needed if you use CMAKE_INSTALL_PREFIX?

File: recipes/gemmi/build.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+mkdir -p build
+cd build
```
Comment (bgruening) [inline recipes/gemmi/build.sh]: pushd could be used here

File: recipes/gemmi/build.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+cmake .. \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release
+
+make -j$CPU_COOUNT
+
+cp gemmi "$PREFIX/bin/"
+
+cd $SRC_DIR
```
Comment (bgruening) [inline recipes/gemmi/build.sh]: popd

**Comment by bgruening** [comment]
Thanks @mcs07! If you really need Windows build conda-forge is fine. Thanks for your contribution.

@maartenbreddels thanks for the review!

File: recipes/gemmi/build.sh (hunk 2 above)
Comment (mcs07) [reply recipes/gemmi/build.sh]: There's no `install` command in the cmake config, so it has to be done manually. So `CMAKE_INSTALL_PREFIX` probably isn't doing anything, I just added it out of habit.

File: recipes/gemmi/meta.yaml (hunk 1 above)
Comment (mcs07) [reply recipes/gemmi/meta.yaml]: Can you explain why? My understanding is that it is required by the resulting executable. The build output contains the following:

    INFO (gemmi,bin/gemmi): Needed DSO lib/libz.so.1 found in conda-forge::zlib-1.2.11-h14c3975_1004

Comment (bgruening) [reply recipes/gemmi/meta.yaml]: zlib should have a run_export definded. It does not hurt to add it, but it should not be needed imho. https://python-packaging-tutorial.readthedocs.io/en/latest/conda.html#run-exports

Comment (mcs07) [reply recipes/gemmi/meta.yaml]: Ah, nice!

**Comment by marcelotrevisani** [comment]
The error on azure windows is not related to your recipe, just a problem in our CI
